spec/lucky_flow_spec.cr

```diff
@@ -16,6 +16,14 @@ describe LuckyFlow do
     flow.el("@heading").text.should eq "Home"
   end
 
+  it "gives a recommendation if element not found" do
+    flow = visit_page_with "<span flow-id='heading'>Home</span>"
+
+    expect_raises LuckyFlow::ElementNotFoundError, "@heading" do
+      flow.el("@harding").displayed?
+    end
+  end
+
```

Comment:
I think a couple other test cases would be good too:

* When the match is far away it should not have a "did you mean". Example `@headboard` should probably not have a `did you mean`
* When the flow-id is correct but the text is not, it should not show `did you mean`
* When negating it should not show `did you mean`. This might be better as a unit test on the `ErrorMessageWhenNotFound` so you can pass in a negation and see that `did you mean` is not included. 

There may be other, but this is all I can think of for now